test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -67,8 +74,12 @@ if [ ! ${ZCC_TEST_NAME[$FRAMEWORK]} ]; then
     exit 1
 fi
 if [[ ${FRAMEWORK} == "tensorflow2" ]]; then
-    if [ "$(printf '%s\n' "0.8.1" "$SMDEBUG_RELEASE_TAG" | sort -V | head -n1)" = "0.8.1" ]; then
-        timeout 720s python mock/tests/zero_code_change/test_${FRAMEWORK}_gradtape_integration.py 2>&1
+    if [ "$(printf '%s\n' "0.7.2" "$SMDEBUG_RELEASE_TAG" | sort -V | head -n1)" = "0.7.2" ]; then
+        if [ "$(printf '%s\n' "0.8.0" "$SMDEBUG_RELEASE_TAG" | sort -V | head -n1)" = "0.8.0" ]; then
```

Comment:
Why are we testing tag value equal to 0.8.0 instead of 0.8.1 ?

Reply:
It is version comparison logic, to judge if version is later than 0.8.0, because path change since 0.8.0